lib/myUser.class.php

```diff
@@ -207,6 +211,66 @@ public function listGroups()
     }
   }
 
+  /**
+   * Checks whether or not a user, based on security.yml settings, has access
+   * to a module's action
+   *
+   * @param string $module  Name of module to check
+   * @param string $action  Name of action to check
+   *
+   * @return boolean
+   */
+  public function checkModuleActionAccess($module, $action)
+  {
+    // Set security property to module's security configuration
+    $securityFilePath = 'modules/'. $module .'/config/security.yml';
+    if ($file = sfContext::getInstance()->getConfigCache()->checkConfig($securityFilePath, true))
+    {
+      require($file);
```

Comment:
Is this loading the config. file inside `$this->security`? It'd be a lot clearer with `sfYaml::load()`.

Reply:
It's using a mechanism, used in other permission processing in Symfony, that returns the filename of a PHP file if it has already been parsed from YAML, so it should be slightly more performant.

Reply:
Will do... thanks!